services/configuration_repository_service.rb

```diff
@@ -157,13 +157,15 @@ def remote_file_a_json_array?(file_path)
     #
     # @return [String]
     def repo_name
+      return "" unless FastlaneCI.env.repo_url
       FastlaneCI.env.repo_url.split("/").last
     end
 
     # The short-form of the configuration repository URL `ueser/repo`
     #
     # @return [String]
     def repo_shortform
+      return "" unless FastlaneCI.env.repo_url
       FastlaneCI.env.repo_url.split("/").last(2).join("/")
```

Comment:
Coming in late here, but 2 things:
1) missing a return statement
2) Maybe it could be rewritten to be `return FastlaneCI.env.repo_url ? FastlaneCI.env.repo_url.split("/").last(2).join("/") : ""` 

but this is totally stylistic ^

Reply:
style wise, I prefer the "guard" statement with an early return over the ternary operator.

Reply:
Great point @snatchev, definitely prefer what is idiomatic, I'm still learning those bits about Ruby 👍

Reply:
That's so Swifty's `if let` 🤣

Reply:
We have this pattern [here](https://github.com/fastlane/ci/blob/master/shared/models/github_provider_credential.rb#L31-L32), so I thought it was cool to use it [here](https://github.com/fastlane/ci/blob/master/services/configuration_repository_service.rb#L160-L161) as well. I agree with @snatchev that the guard statement seems more like idiomatic ruby, but I'm open to changing it to the normal `if/else` statement if it's more readable.

Reply:
Also, I included the extra return in my [explicit return PR](https://github.com/fastlane/ci/pull/325/files#diff-51bddc1ca8b9376a96ac62cf51d8814fR161). I kept them separate but worked on them both at the same time.